data/snippet/uuid.toml

```diff
@@ -0,0 +1,42 @@
+id = "uuid"
+name = "Create UUIDs"
+description = "Create a version 4 revision 2 UUID inside a worker"
+code = '''
+async function handleRequest(request) {
+  const UUID = createUUID()
+  const obj = {id: UUID}
```

Comment:
This hasn't been through prettier :(

Reply:
I misunderstood our conversation at devweek. I thought you had said it was optional, which had confused me, but I will note that it is not optional for the future

Reply:
This will be easier since we are using the template registry now. I will port it over for you once this is reviewed.

Or even better if you could port this to now use the template registry https://github.com/victoriabernard92/template-registry